DOCUMENTATION.md

```diff
@@ -4388,7 +4388,7 @@ The name must be a string; the optional doc argument can have any type.
     "release_image": Optional[List[ReleaseImageType]],
     "should_be_running_on__application": Optional[List[TypeApplication]],
     "is_running_on__device": Optional[List[TypeDevice]],
-    "should_be_running_on__device": Optional[List[TypeDevice]],
+    "is_pinned_to__device": Optional[List[TypeDevice]],
```

Comment:
The commit name is `v7 model: Replace the device.should_be_running__release with is_pinned_on__release`, is one of these a mistake?

Reply:
I guess we could do a separate commit strictly speaking.
In the js SDK I chose to update each term in a separate commit, so I grouped both the normal fact type (`device.is_pinned_on__release`) and its new synonymous form (`release.is_pinned_to__device`) in the same commit.

Reply:
I added both in the same commit as a pattern. In the node one we sometimes did it together and sometimes spplited. I decided to make a single commit entry for adding both the regular form and the synonymous form, I think it makes more clear to read and creates less unnecessary commit entries.